### PR TITLE
Linked to the contributions guide in readme

### PR DESCRIPTION
**Pull request by def0x099**
see commits :)

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 This repo contains the source material and build tools for: http://www.yaml.org
 
 The upkeep of http://www.yaml.org is a community project. You can easily
-contribute! See [[Contributing.md]] for details.
+contribute! See [the Contribution guide](./Contributing.md) for details.
```
Comment (Qix-) [inline ReadMe.md]: Don't do `./`, just do `Contributing.md` :) It helps sites other than Github render content a bit better (sounds strange, I know).

```suggestion
contribute! See [the Contribution guide](Contributing.md) for details.
```

**Comment by def14nt** [comment]
@Qix- okok lemme crunch down on my dinner and I'll have a look :)

**Comment by def14nt** [comment]
@Qix- Better?

**Comment by def14nt** [comment]
@Qix- You gonna merge this?

**Pull request merged**